pkgs/native_assets_builder/test_data/use_all_api/pubspec.yaml

```diff
@@ -0,0 +1,20 @@
+name: simple_link
+description: Build 4 assets for linking, only keep two in link.dart.
```

Comment:
This seems not correct, the linker adds all assets it gets to it's output. Too much Copy&Paste? :D